master/docs/developer/cls-forcesched.rst

```diff
@@ -1,128 +0,0 @@
-.. -*- rst -*-
-.. _ForceScheduler:
-
-ForceScheduler
---------------
-
-The force scheduler has a symbiotic relationship with the web status, so it deserves some further description.
-
-Parameters
```

Comment:
@tardyp I thought you updated and moved these docs to http://docs.buildbot.net/latest/manual/cfg-schedulers.html#forcescheduler-scheduler

Will you update this file?